state-network.md

```diff
@@ -255,53 +269,297 @@ sedes      := Container(enrs: List[byte_list, max_length=32], payload: byte_list
 > A response with an empty `payload` and empty `enrs` indicates that the node is not aware of any closer nodes, *nor* does the node have the requested content.
 
 
-#### Advertise (0x07)
+#### Offer (0x07)
 
-Advertise a set of content keys that this node has proofs available for.
+Offer a set of content keys that this node has proofs available for.
 
 ```
 message_id := 7
 type       := request
-sedes      := List[byte_list, max_length=32]
+sedes      := List[byte_list, max_length=64]
```

Comment:
What is the idea behind the `max_length` change from 32 to 64?

I assume it would be up to the implementations to make sure that you limit adding keys to the `List` to not go over your preferred UDP packet size. But even with the smallest content keys (e.g. 53 bytes for https://github.com/ethereum/stateless-ethereum-specs/blob/piper/state-network-revised/state-network.md#account-trie-proof), you'd be going far over the typical MTU?

Of course being able to offer a higher amount of content is better for having longer lived uTP streams, I'm just not fully understanding how the current limits are supposed to work.

Reply:
Nothing compelling here.  Easy future proofing in the case where we end up with content keys that are short enough to fit more in.  You are correct hat implementations will still need to ensure that the payload size doesn't exceed the UDP packet size.

In practice, I think most of the time we'll be sending only a very small number of items over these payloads (often just one)